tests/foreman/api/test_permission.py

```diff
@@ -205,14 +200,13 @@ def give_user_permission(self, perm_name):
         self.user = self.user.update(['role'])
 
     @tier1
-    def test_create(self):
+    def test_positive_create_permission(self):
```

Comment:
It's not that clear too. `test_positive_create` means we're creating something. Here we check whether 'create' permission works (user can create entities with this permission and can't without it). Not sure how better to describe that in test name. Maybe you have ideas?
Same for `read`/`delete` etc.